VolleyPaint/Assets/Scripts/Editor/IntReferenceDrawer.cs

```diff
@@ -0,0 +1,57 @@
+using UnityEngine;
+using UnityEditor;
+
+[CustomPropertyDrawer(typeof(IntReference))]
+public class IntReferenceDrawer : PropertyDrawer {
```

Comment:
what are all the IntReferenceDrawer/IntReference/IntVariable scripts for?

Reply:
They are not used yet. 
We can probably delete it for now. 
I was planning to use them for any kind of integers that can be used for players or game states or number of ability chances or something like that.